addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -371,3 +371,7 @@ $pagination-padding-x-lg: $pagination-padding-x !default;
 
 $pagination-border-radius-lg: $pagination-border-radius !default;
 $pagination-border-radius-sm: $pagination-border-radius !default;
+
+// Nav and tabs
+$nav-tabs-link-active-color: $link-color;
+$nav-underline-link-active-color: $link-color;
```

Comment:
Not sure about these changes 🤔
Bootstrap default (black) makes more sense to me since you're not supposed to click active elements.
Moreover it would invalidate ant user customization made through the SCSS editor... 

Were these changes need for aesthetic purposes only? If it's the case I'd suggest to drop them